dumb-init.c

```diff
@@ -42,6 +42,7 @@ int signal_rewrite[MAXSIG + 1] = {[0 ... MAXSIG] = -1};
 pid_t child_pid = -1;
 char debug = 0;
 char use_setsid = 1;
+char forward_sleep = 10;
```

Comment:
How about `20` seconds to be consistent with what we have in deployment scripts?

Reply:
I've considered it but AFAIK 20s was an arbitrary value, not related to any real limitation and it seemed to much.